azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.yml

```diff
@@ -14,7 +14,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Get extension properties'
   code: |-
-    PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84"
+    PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```

Comment:
Object ID not updated in the output.

Reply:
Changes implemented here are ok for the purposes of this PR